lib/parseURL.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const { URL } = require('url')
+
+const BASE_URL = 'http://localhost:80'
+
+/**
+ * Create URL from `options` object
+ *
+ * @param {Object} options
+ * @param {(Object|String)} options.url
+ * @param {Object} [options.query]
+ * @return {URL}
+ */
+module.exports = function parseURL (options) {
+  let parsedURL = null
+  let mergedQuery = {}
+
+  if (typeof options.url === 'object') {
```

Comment:
Is this case possible? From the README, we are not supporting this.

Reply:
According the schema, `url` may be `String` or `Object`.